paper/paper.bib

```diff
@@ -157,30 +157,30 @@ @article{schulenburg2017
 }
 
 @misc{vectorclass,
-  author = {Fog, Agner},
-  title = {{VCL:} {C++} vector class library},
-  year = {2020},
-  publisher = {GitHub},
-  journal = {GitHub repository},
-  howpublished = {\url{https://github.com/vectorclass/version2}},
-  commit = {46a58414bc6128b61d77186656860ca7b0b961d8},
+ author = {Fog, Agner},
+ title = {{VCL:} {C++} vector class library},
+ year = {2020},
+ publisher = {GitHub},
+ journal = {GitHub repository},
+ howpublished = {\url {https://github.com/vectorclass/version2}},
+ commit = {46a58414bc6128b61d77186656860ca7b0b961d8},
 }
 
 @misc{qsl2021,
-  author = {Astrakhantsev, Nikita},
-  title = {QSL\_at\_QC},
-  year = {2021},
-  publisher = {GitHub},
-  journal = {GitHub repository},
-  howpublished = {\url{https://github.com/nikita-astronaut/QSL\_at\_QC}},
+ author = {Astrakhantsev, Nikita},
+ title = {{QSL\_at\_QC}},
+ year = {2021},
+ publisher = {GitHub},
+ journal = {GitHub repository},
+ howpublished = {\url {https://github.com/nikita-astronaut/QSL\_at\_QC}},
```

Comment:
```suggestion
 howpublished = {\url{https://github.com/nikita-astronaut/QSL\_at\_QC}},
```
Am I missing why these spaces are being added?

Reply:
betterbib is at fault here. I guess it's an issue with their autoformatter, but since LaTeX still generated proper pdfs I ignored the spaces. Fixed now.